extra_docs/jfs/README.md

```diff
@@ -0,0 +1,209 @@
+# Jexia Fileset Module (JFS)
+
+JFS is used for the uploading, fetching and manipulating files as any other user data. Basically, fileset has the same 
+functionality as dataset, but provides a way to maintain large data objects (files).  
+Before uploading any file you need to manually create a fileset with web management application 
+[app.jexia.com](https://app.jexia.com).
+You can also set AWS bucket and credentials so that your files will be automatically uploaded to the Amazon storage.
```

Comment:
suggestion: _automatically uploaded to ~~the~~ **YOUR** Amazon storage_

Reply:
👍